messages/nl/app.php

```diff
@@ -0,0 +1,200 @@
+    'Are you sure you want to delete this review?' => 'Weet je zeker dat je deze recensie wilt verwijderen?',
+    'Bad Request' => 'Foute Verzoek',
+    'Be positive, constructive and respectful' => 'Weet positief, constructief en respectvol',
+    'Bump' => 'Bump',
+    'Bumped At' => 'Gebumpt Op',
+    'Bumped {time}' => 'Gebumpt {time}',
+    'Bumped recently' => 'Recent gebumpt',
+    'Cancel' => 'Annuleer',
+    'Categorize your server by keywords (e.g. game title)' => 'Categoriseer je server met sleutelwoorden (bijv. spel titel)',
+    'Category' => 'Categorie',
+    '<b>Categories</b> and <b>Popular tags</b>' =>  '<b>Categorieën</b> en <b>Populaire sleutelwoorden</b>',
+    'Copied the URL' => 'URL gekopieerd',
+    'Could not bump the server because the invite link is invalid (expired?)' => 'Kon de server niet bumpen omdat de uitnodiging link ongeldig is (verlopen?)',
+    'Could not delete the server' => 'Kon de server niet verwijderen',
+    'Could not delete the review' => 'Kon de recensie niet verwijderen',
+    'Created at {datetime}' => 'Gemaakt op {datetime}',
+    'Dashboard' => 'Controlepaneel',
+    'Descending' => 'Aflopend',
+    'Description' => 'Beschrijving',
+    'Delete server' => 'Verwijder server',
+    'Delete reply' => 'Verwijder reactie',
+    'Delete review' => 'Verwijder recensie',
+    'Discord servers partnered with <span class="brand">DISBOARD</span>' => 'Discord servers die partners zijn met <span class="brand">DISBOARD</span>',
+    'Discord Server List' => 'Discord Server Lijst',
+    'Discord Server Reviews' => 'Discord Server Recensies',
+    'Discord Servers Bumped Recently' => 'Recentelijke Gebumpte Discord Servers',
+    'Discord servers categorized as <strong>{category}</strong>' => 'Discord servers gecategoriseerd als <strong>{category}</strong>',
+    'Discord servers tagged with {tag}' => 'Discord servers getagd met {tag}',
+    '<span class="discord-logo">Discord</span> Servers' => '<span class="discord-logo">Discord</span> Servers',
+    'Discord servers tagged with <strong>{tag}</strong>' => 'Discord servers getagd met <strong>{tag}</strong>',
+    'Don\'t forget to check "Set this link to never expire"' => 'Vergeet niet om op "Laat de link nooit verlopen" te klikken',
+    'Edit' => 'Bewerk',
+    'Edit Server' => 'Bewerk Server',
+    'Enter keyword(s)' => 'Voer trefwoord(en)',
+    'Failed to Sync' => 'Synchroniseren Mislukt',
+    'Games' => 'Spellen',
+    'Gaming' => 'Gaming',
+    'Go to Dashboard' => 'Ga naar het controlepaneel',
+    'Go Back' => 'Ga Terug',
+    'Go Home' => 'Ga naar Startpagina',
+    'Home' => 'Startpagina',
+    'If you don\'t add a bot, you can NOT present various informations about your server, create an invitation link or bump up your servers with commands' => 'Als je de bot niet toevoegt kunt je NIET bepaalde informatie zien over je server, een directe uitnodiging link maken of je server bumpen met commando''s.',
```

Comment:
kunt -> kun

Part of me wants to shove NIET all the way to the back where it technically belongs, but this conveys the message better.